library/core/src/fmt/mod.rs

```diff
@@ -333,7 +333,7 @@ pub struct Arguments<'a> {
 #[unstable(feature = "fmt_internals", issue = "none")]
 impl<'a> Arguments<'a> {
     #[inline]
-    #[rustc_const_unstable(feature = "const_fmt_arguments_new", issue = "none")]
+    #[rustc_const_stable(feature = "const_fmt_arguments_new", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
You don't need to const stabilize this to use it internally. You should be able to mark whatever spans as `allow_internal_unstable` or similarly with [`Span::mark_with_reason`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_span/span_encoding/struct.Span.html#method.mark_with_reason).

Reply:
Thank you for the feedback.